airbyte-ci/connectors/connector_ops/connector_ops/utils.py

```diff
@@ -253,22 +257,20 @@ def language(self) -> ConnectorLanguage:
             return ConnectorLanguage.LOW_CODE
         if Path(self.code_directory / "setup.py").is_file() or Path(self.code_directory / "pyproject.toml").is_file():
             return ConnectorLanguage.PYTHON
-        try:
-            with open(self.code_directory / "Dockerfile") as dockerfile:
-                if "FROM airbyte/integration-base-java" in dockerfile.read():
-                    return ConnectorLanguage.JAVA
-        except FileNotFoundError:
-            pass
+        if Path(self.code_directory / "build.gradle").is_file():
+            return ConnectorLanguage.JAVA
         return None
 
     @property
-    def version(self) -> str:
+    def version(self) -> Optional[str]:
```

Comment:
why is this one optional? I get becuase `self.version_in_dockerfile_label` could now be `None` - but shouldn't we error out if we can't retrieve a version for the connector?